src/service/url-replacements-impl.js

```diff
@@ -663,19 +663,6 @@ export class GlobalVariableSource extends VariableSource {
     this.setAsync('MAKE_BODY_VISIBLE', () => {
       return Services.performanceFor(win).getMakeBodyVisible();
     });
-
-    this.setAsync('AMP_STATE', key => {
```

Comment:
want to confirm with @choumx that this is fine. 
Do we see AMP_STATE used in other cases except analytics pings? form request maybe.

Reply:
This was originally added for AoG which uses amp-state extensively. E.g. might be used for forms or other data endpoints as you suggest.

I'm not sure if it's actively used today though. We can add a `dev().expectedError` to find usage.

Reply:
SG, will revisit after we collect some data on its usage.